src/main/python/janusgraph_python/core/attribute/TextPredicate/Text.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from gremlin_python.process.traversal import P
+
+
+class Text(object):
+    """
+    Class implementing Gremlin Python's Predicate class, and extending it for JanusGraph based Text Predicates.
+    """
+
+    @staticmethod
+    def textContains(value):
+        """
+        Implements JanusGraph's textContains functionality
+
+        Args:
+            value (str):
+
+        Returns:
+            P
+        """
+
+        predicate = P("textContains", value)
```

Comment:
It's just a nit and I think @mbrukman already commented on this somewhere, but I would also remove these temp variables like `predicate` and instead directly `return P("textContains", value)`. This increases readability in my opinion as it doesn't involve another variable that isn't really needed.
It's also a common refactoring called _Inline Temp_, see [[1]](https://refactoring.guru/inline-temp) and [[2]](https://refactoring.com/catalog/inlineTemp.html).

Reply:
I've done that in fresh commit for both Text and Geo classes